website/docs/d/space.html.markdown

```diff
@@ -1,49 +1,49 @@
-
-## Example Usage
-
-```hcl
-data "ibm_space" "spaceData" {
-  space = "prod"
-  org   = "someexample.com"
-}
-```
-
-The following example shows how you can use the data source to reference the space ID in the `ibm_service_instance` resource.
-
-```hcl
-resource "ibm_service_instance" "service_instance" {
-  name              = "test"
-  space_guid        = "${data.ibm_space.spaceData.id}"
-  service           = "cloudantNOSQLDB"
-  plan              = "Lite"
-  tags              = ["cluster-service", "cluster-bind"]
-}
-
-```
-
-## Argument Reference
-
-The following arguments are supported:
-
-* `org` - (Required) The name of your Bluemix org. The value can be retrieved by running the `bx iam orgs` command in the [Bluemix CLI](https://console.ng.bluemix.net/docs/cli/reference/bluemix_cli/index.html#getting-started).
-* `space` - (Required) The name of your space. The value can be retrieved by running the `bx iam spaces` command in the Bluemix CLI.
-
-## Attributes Reference
-
-The following attributes are exported:
-
-* `id` - The unique identifier of the space.  
-* `managers` - The emails (associated with IBM ID) of the users who have manager role in this space
-* `auditors` - The emails (associated with IBM ID) of the users who have auditor role in this space
-* `developers` - The emails (associated with IBM ID) of the users who have developer role in this space
+---
+layout: "ibm"
+page_title: "IBM: ibm_space"
+sidebar_current: "docs-ibm-datasource-space"
+description: |-
+  Get information about an IBM Bluemix space.
+---
+
+# ibm\_space
+
+Import the details of an existing IBM Bluemix space as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration by using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_space" "spaceData" {
+  space = "prod"
+  org   = "someexample.com"
+}
+```
+
+The following example shows how you can use the data source to reference the space ID in the `ibm_service_instance` resource.
+
+```hcl
+resource "ibm_service_instance" "service_instance" {
+  name              = "test"
+  space_guid        = "${data.ibm_space.spaceData.id}"
+  service           = "cloudantNOSQLDB"
+  plan              = "Lite"
+  tags              = ["cluster-service", "cluster-bind"]
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Required) The name of your Bluemix organization. You can retrieve the value by running the `bx iam orgs` command in the [Bluemix CLI](https://console.ng.bluemix.net/docs/cli/reference/bluemix_cli/index.html#getting-started).
+* `space` - (Required) The name of your space. You can retrieve the value by running the `bx iam spaces` command in the Bluemix CLI.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the space.  
+* `managers` - The email addresses (associated with IBM ID) of the users who have a manager role in this space.

```

Comment:
We're using the casing `IBMid` for accounts. https://www.ibm.com/account/us-en/signup/register.html